docs/egress-firewall.md

```diff
@@ -0,0 +1,35 @@
+# EgressFirewall
+
+## Introduction
+
+The egressFirewall feature enables a cluster administrator to
+limit the external hosts that a pod in a project can access.
+The EgressFirewall object rules apply to all pods that share
+the namespace with the egressfirewall object. A namespace only
+supports having one EgressFirewallObject.
+
+## Example
+
+The yaml  below is an example of a simple egressFirewall object
+
+```yaml
+kind: EgressFirewall
+apiVersion: k8s.ovn.org/v1
+metadata:
+  name: default
+  namespace: default
+spec:
+  egress:
+  - type: Allow
+    to:
+      cidrSelector: 1.2.3.0/24
+  - type: Deny
+    to:
+      cidrSelector: 0.0.0.0/0
+```
+
+this example would allow pods in the default namespace to connect to
```

Comment:
Some thoughts, not entirely finalized. I'll have to explain the same concept in the product docs soon enough.

> This example allows Pods in the default namespace to connect to any external host within the range 1.2.3.0 to 1.2.3.255 and denies traffic to all other external hosts.

> The priority of a rule is determined by its placement in the `egress` array. An earlier rule is processed before a later rule. In the previous example, if the rules are reversed, all traffic is denied, including any traffic to hosts in the 1.2.3.0/24 CIDR block.

Reply:
changed to your version